networking/cidr-range-definitions.adoc

```diff
@@ -35,12 +43,31 @@ endif::[]
+In the Service CIDR field, you must specify the IP address range for services. 
+ifdef::openshift-rosa,openshift-dedicated[]
+It is recommended, but not required, that the address block is the same between clusters. This will not create IP address conflicts. 
+endif::openshift-rosa,openshift-dedicated[]
+The range must be large enough to accommodate your workload. The address block must not overlap with any external service accessed from within the cluster. The default is `172.30.0.0/16`.
 
 [id="pod-cidr-description"]
 == Pod CIDR
-In the pod CIDR field, you must specify the IP address range for pods. It is recommended, but not required, that the address block is the same between clusters. This will not create IP address conflicts. The range must be large enough to accommodate your workload. The address block must not overlap with any external service accessed from within the cluster. The default is `10.128.0.0/14`.
+In the pod CIDR field, you must specify the IP address range for pods. 
+
+ifdef::openshift-enterprise[]
+The pod CIDR is the same as the `clusterNetwork` CIDR and the cluster CIDR. For more information about configuring CIDR for these, see xref:../networking/cluster-network-operator.adoc#nw-operator-cr-cno-object_cluster-network-operator[Cluster Network Operator Configuration] and the xref:../networking/configuring-cluster-network-range.adoc#configuring-cluster-network-range[Configuring the cluster network range]. 
+endif::openshift-enterprise[]
+ifdef::openshift-rosa,openshift-dedicated[]
+It is recommended, but not required, that the address block is the same between clusters. This will not create IP address conflicts. 
+endif::openshift-rosa,openshift-dedicated[]
+The range must be large enough to accommodate your workload. The address block must not overlap with any external service accessed from within the cluster. The default is `10.128.0.0/14`.
 
 [id="host-prefix-description"]
 == Host Prefix
-In the Host Prefix field, you must Specify the subnet prefix length assigned to pods scheduled to individual machines. The host prefix determines the pod IP address pool for each machine. For example, if the host prefix is set to `/23`, each machine is assigned a `/23` subnet from the pod CIDR address range. The default is `/23`, allowing 512 cluster nodes, and 512 pods per node (both of which are beyond our maximum supported).
+In the Host Prefix field, you must specify the subnet prefix length assigned to pods scheduled to individual machines. The host prefix determines the pod IP address pool for each machine.  
+ifdef::openshift-rosa,openshift-dedicated[]
+For example, if the host prefix is set to `/23`, each machine is assigned a `/23` subnet from the pod CIDR address range. The default is `/23`, allowing 512 cluster nodes, and 512 pods per node (both of which are beyond our maximum supported).
```

Comment:
@zhaozhanqi The 512 cluster nodes only applies to ROSA and OSD documentation. In the preview for OpenShift enterprise, the correct 510 cluster notes appears:
![image](https://github.com/openshift/openshift-docs/assets/77501157/3799f03c-4659-4cbe-b9f6-6f5e71a85e80)

The `idfef` and `endif` tags are to help with sharing this information across the different documentation sets, since some information overlaps but some information is different. The effect is that lines 71-73, with the "510 cluster nodes", only displays on docs.openshift.com, whereas lines 67-69 display in the ROSA documentation.